docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -289,8 +342,13 @@ Step 1: Configure the Network on the First Node
 
         --peers tcp://172.0.1.3,tcp://172.0.1.4
 
-#. If the XO transaction processor is not running, start it as described in
-   Step 7 of :doc:`docker`.
+#. If necessary, start the other Sawtooth components.
+
```

Comment:
settings-tp must be started as well.

Reply:
Answering my own question: It's clearer to stop all Sawtooth components in step 1, then restart them all here. That allows me to add a note that settings-tp and poet-validator-registry-tp are required, and that intkey-tp-python and xo-tp-python are "optional".